src/components/timeout-dialog/_timeout-dialog.scss

```diff
@@ -16,27 +16,24 @@
   -moz-opacity: 0.8;
 }
 
-html {
-  &.no-scroll {
-    overflow: hidden;
-  }
-}
-
```

Comment:
normally - with styles like this which are global, I'd say leave it in _just in case_ - but in this case it's not seemingly ever added to anything (other than one prototype in github org) https://github.com/search?q=org%3Ahmrc+no-scroll&type=code&p=1 

I know we have other orgs we don't know about using our stuff, but in this case it does seem really unused / legacy thing